src/components/SigninForm/index.js

```diff
@@ -15,28 +14,27 @@ const Signin = () => {
 
   return (
     <React.Fragment>
-      <div className="signStr spacebottom">Login to your account</div>
+      <div className='signStr spacebottom'>Login to your account</div>
       <form>
         <InputBox
-          type="email"
-          name="email"
-          placeholder="EMAIL"
+          type='email'
+          name='email'
+          placeholder='EMAIL'
           onChangeInput={e => onChangeEmail(e)}
         />
         <InputBox
-          type="password"
-          name="password"
-          placeholder="PASSWORD"
+          type='password'
+          name='password'
+          placeholder='PASSWORD'
           onChangeInput={e => onChangePassword(e)}
         />
-        <div className="forgot">
+        <div className='forgot'>
           <span onClick={() => alert('Open dialog')}>Forgot password?</span>
```

Comment:
Why is there an alert statement here?

Reply:
It's not yet implemented, We were just testing for the click event